rust/src/delta.rs

```diff
@@ -109,6 +110,8 @@ pub enum DeltaTableError {
     PartitionError { partition: String },
     #[error("Invalid partition filter found: {}.", .partition_filter)]
     InvalidPartitionFilter { partition_filter: String },
+    #[error("Invalid retention period, retention for Vacuum must be superior than 7 weeks.")]
```

Comment:
```suggestion
    #[error("Invalid retention period, retention for Vacuum must be greater than 1 week (168 hours)")]
```

I believe this was a typo